tests/test_parser.py

```diff
@@ -129,3 +129,26 @@ def test_structure_names(cs: cstruct) -> None:
     assert cs.c.__name__ == "c"
     assert cs.d.__name__ == "c"
     assert cs.e.__name__ == "e"
+
+
+def test_structure_names(cs: cstruct) -> None:
+    cdef = """
+    /* Standard libs */
+    #include <stdint.h> // defines fixed data types: int8_t...
+    /* user libs */
+    #include "myLib.h"  // my own header
+
+    typedef struct myStruct
+    {
+        char charVal[16];
+    }
+    """
+    cs.load(cdef)
+
+    assert cs.includes[0] == "<stdint.h>"
+    assert cs.includes[1] == "\"myLib.h\""
```

Comment:
With a strip or change in the regex pattern I meant to remove the quotes from the result, so that this would be `myLib.h` instead of `"myLib.h"`.